openstack/blockstorage/extensions/volumeactions/results.go

```diff
@@ -50,6 +55,101 @@ func (r commonResult) Extract() (map[string]interface{}, error) {
 	return s.ConnectionInfo, err
 }
 
+// VolumeImage contains information about volume upload to an image service.
```

Comment:
Because it's in the `volumeactions` package, it seems like at least one of the `Volume`s in the type are redundant. Are `VolumeImageType` and `ImageVolumeType` both misleading/inaccurate?